README.md

```diff
@@ -21,7 +21,7 @@ This app is intended to be used on an offline computer by an election admin.
    - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
    - approved official ballots in pdf format:
      - one for each combination of ballot style and precinct
-     - file name pattern: `election-${electionHash}-ballot-style-${ballotStyle}-precinct-id-${precinctId}.pdf`
+     - file name pattern: `election-${electionHash}-ballot-style-${ballotStyleId}-precinct-${dashify(precinctName)}-${precinctId}.pdf`
```

Comment:
Do we need to prefix each variable with a key/label?

Reply:
🤷 it does help when the values are gibberish like `77`.